installation/cmd/run.ps1

```diff
@@ -10,14 +9,9 @@ $CURRENT_DIR = Split-Path $MyInvocation.MyCommand.Path
 $SCRIPTS_DIR = "${CURRENT_DIR}\..\scripts"
 $DOMAIN = "kyma.local"
 $INSTALLER_EXTRA_ARGS = ""
-$MINIKUBE_EXTRA_ARGS = ""
+$MINIKUBE_EXTRA_ARGS = "-memory 10240 -disk_size 30g"

```

Comment:
I'd simplify it and add those args directly to `minikube start` [here](https://github.com/kyma-project/kyma/blob/master/installation/scripts/minikube.ps1#L42).